code/modules/economy/traders/generic.dm

```diff
@@ -46,15 +46,14 @@
 /datum/commodity/trader/generic/sheetmetal
 	comname = "Sheet Metal"
 	comtype = /obj/item/sheet/steel
-	amount = 200
```

Comment:
accidental removal?

Reply:
Not accidental, it seemed VERY pointless to have a crate option for 200 sheets at a click of a button, but having a chance based trader item be limited to the amount of a single "200 sheet crate" over several purchases doesn't seem particularly worth having. At that point it just felt like filler, even with my change. So I just made it unlimited, unless I messed up making it actually unlimited by accident.